airflow-core/src/airflow/api_fastapi/core_api/datamodels/ui/config.py

```diff
@@ -24,13 +24,13 @@
 class ConfigResponse(BaseModel):
     """configuration serializer."""
 
-    page_size: int
-    auto_refresh_interval: int
-    hide_paused_dags_by_default: bool
+    page_size: int | None
```

Comment:
Aren't these all meant to have defaults in the configs, so they shouldn't actually be null, right @pierrejeambrun